src/OpenConextMonitorBundle.php

```diff
@@ -18,14 +18,14 @@
 
 namespace OpenConext\MonitorBundle;
 
-use OpenConext\MonitorBundle\DependencyInjection\Compiler\HealthCheckPass;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
-use Symfony\Component\HttpKernel\Bundle\Bundle;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+use Symfony\Component\HttpKernel\Bundle\AbstractBundle;
 
-class OpenConextMonitorBundle extends Bundle
+class OpenConextMonitorBundle extends AbstractBundle
 {
-    public function build(ContainerBuilder $container): void
+    public function loadExtension(array $config, ContainerConfigurator $container, ContainerBuilder $builder): void
     {
-        $container->addCompilerPass(new HealthCheckPass());
+        $container->import('../config/services.yaml');
```

Comment:
Safe to use a relative path here? Maybe use a param to refer to the config?

`kernel.project_dir`

Reply:
BTW, kernel.projectDir refers to the consuming project. The path here is relative to the bundle (hence the move from Resources to the /config of the bundle structure

![image](https://github.com/OpenConext/Monitor-bundle/assets/7671998/3208b904-ab86-4370-bead-6708dc5043a4)

Reply:
Thanks for your explanation